src/client.rs

```diff
@@ -563,6 +563,20 @@ impl <T: Read + Write> Session<T> {
         self.run_command_and_check_ok(&format!("UID COPY {} {}", uid_set, mailbox_name))
     }
 
+    /// Moves each message in the sequence into the destination mailbox.
+    pub fn imap_move(&mut self, sequence_set: &str, mailbox_name: &str) -> Result<()> {
```

Comment:
`move` being reserved is unfortunate here, it's true... Perhaps `mv`?

Reply:
I don't have a strong preference either way. It will still remain the only function that does not match the IMAP command sadly...

Reply:
Renamed to mv